plugins/extractors/elastic/elastic.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"reflect"
+	"sort"
```

Comment:
@GrayFlash if it is to solve the test problem. why don't we just sort the expected and the actual data in test?

Reply:
that actually works too 🤔

Reply:
Thought we should maintain the consistency in order of records being emmited. Usually all other extractor's client follow a certain pattern in showing the result of query.

Reply:
Hmm but if the problem is in the test case (flaky test) and there is no requirements to sort the extracted records, adding this in the main code would just add some additional unnecessary computation. 🤔

Reply:
@mabdh was able to fix it for indexes actually in the test part, but I can seem to find a function to get the data in `Schema` from records. There doesn’t seem to be any function in metadata model interface for it. So it is still being handled by sorting the columns before emitting. Should work?? Or else we might need some changes in proto models itself.

Reply:
please let me know if I'm missing something here. 😅

Reply:
In that case maybe we can add `GetSchema` here? @StewartJingga 
https://github.com/odpf/meteor/blob/5f7cd1471bc68ede73b4e1206856373d84cf654e/models/metadata.go#L11

Reply:
hey, folks any further comment on this? shall I move on with what I suggested?

Reply:
@GrayFlash IMO, if the purpose of the test is not in the `order of columns` or `number of extracted columns should be equal to the number provided columns`. Then it does fine to reduce the dummy data to 1.

Reply:
@StewartJingga @mabdh for now I have removed one column from dummy data and tested it for the same, if we end up adding `GetSchema`, we can fix it other way.